trojsten/contests/migrations/0021_auto_20220912_1414.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 2.2.24 on 2022-09-12 14:14
```

Comment:
Neviem či je to dôležité, ale možno by si mohol premenovať tento file na `0022_...`, v suši PR máme migráciu 0021 a neviem ako sa by sa to poprípade vysporiadalo s dvoma s rovnakým číslom.

Reply:
no keď som to zmenil, tak nezbehnú migrácie

Reply:
Hm tak to bude treba spraviť neskôr keď sa budú mergovať tie PRs, lebo teraz majú v podstate oba PR ako dependency danú migráciu 0020, tak keď sa mergne jeden bude treba ten druhý rebasnúť na main a zmeniť tú migráciu.